.ruby-version

```diff
@@ -1 +1 @@
-2.4.1
+2.4.2
```

Comment:
I'd suggest changing ruby version in a different PR/deploy.

Reply:
👍